### PR TITLE
Display gym/trainer contact info if provided by API

### PR DESCRIPTION
**Pull request by moderndayNeo**

### CODE REVIEW

File: src/PageView/NearMe/NearMeGyms/NearMeGyms.js [hunk 1]
```diff
@@ -1,42 +1,49 @@
...
+    };
     let errorHandle = () => setLocation({ lat: 51.5007, lng: -0.1246 });
     navigator.geolocation.getCurrentPosition(successPosition, errorHandle);
   }, []);
 
   useEffect(() => {
     (async () => {
       const response = await fetch(
-        `https://discover.search.hereapi.com/v1/discover?at=${location.lat},${location.lng}&q=gym&limit=15&apiKey=bYn1_nn2_CYDA29oVvcibuL4otpQL3F0HIF5aBnlcW4`
+        `https://discover.search.hereapi.com/v1/discover?at=${location.lat},${location.lng}&q=${query}&limit=15&apiKey=bYn1_nn2_CYDA29oVvcibuL4otpQL3F0HIF5aBnlcW4`
```
Comment (rmallols) [inline src/PageView/NearMe/NearMeGyms/NearMeGyms.js:27]: The URL is very large, please separate it within tokens, e.g.:

`let endpoint = 'http://...`
`let queryParams` = 'limit=15&...'

File: src/PageView/NearMe/NearMeGyms/NearMeGyms.js [hunk 2]
```diff
@@ -1,42 +1,49 @@
-import React, { useState, useEffect } from 'react';
-import {
-  GoogleMap,
-  LoadScript,
-  Marker,
-} from '@react-google-maps/api';
-import './NearMeGyms.css';
-import InfoWindows from './InfoWindows/InfoWindows'
+import React, { useState, useEffect } from "react";
```
Comment (rmallols) [inline src/PageView/NearMe/NearMeGyms/NearMeGyms.js:1]: This component is huge! Please consider moving some bits out of it by e.g. creating some children components

Comment (moderndayNeo) [reply src/PageView/NearMe/NearMeGyms/NearMeGyms.js]: Thanks for the feedback, Ricardo. Making those changes now

File: src/PageView/NearMe/NearMeGyms/NearMeGyms.js [hunk 3]
```diff
@@ -21,11 +21,13 @@ export default function NearMeGyms() {
     navigator.geolocation.getCurrentPosition(successPosition, errorHandle);
   }, []);
 
+  let endpoint = 'https://discover.search.hereapi.com/v1/discover';
```
Comment (rmallols) [inline src/PageView/NearMe/NearMeGyms/NearMeGyms.js:24]: Much more readable, thanks mate :)

File: src/PageView/NearMe/List/List.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
```
Comment (rmallols) [inline src/PageView/NearMe/List/List.js:1]: Much cleaner! Please use single quotes: https://github.com/airbnb/javascript#strings--quotes

**Pull request merged**